ci_framework/roles/devscripts/tasks/03_install.yml

```diff
@@ -53,6 +53,17 @@
     regexp: "^    address_v4(.+)$"
     replace: "    address_v4: {{ ansible_default_ipv4.address }}"
 
+- name: Copy the CI token file in devscript repo directory
+  when: cifmw_devscripts_ci_token_file is defined
+  tags:
+    - bootstrap
+  ansible.builtin.copy:
+    src: "{{ cifmw_devscripts_ci_token_file }}"
+    dest: "{{ cifmw_devscripts_repo_dir }}/ci_token"
+    owner: "{{ cifmw_devscripts_user }}"
+    group: "{{ cifmw_devscripts_user }}"
+    mode: "0644"
```

Comment:
would probably be safer to set "0400" or, at least, "0600"

Reply:
Done